examples/quickstart/jupyter-notebooks/notebooks/03-query/04-UnionOperations.ipynb

```diff
@@ -0,0 +1,541 @@
+    "\n",
+    "Notice that these results are not in order by time – even though the individual sets did `ORDER BY` time. Druid simply concatenated the two result sets together.\n",
+    "\n",
+    "Optionally, run the next cell – it shows the precise `EXPLAIN PLAN` for the query. You can see there are two `query` execution plans, one for each query, and that Druid's planning process has taken time to optimize how the query above will actually execute."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "97934da2-17d1-4c91-8ae3-926cc89185c1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(json.dumps(json.loads(sql_client.explain_sql(sql)['PLAN']), indent=2))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "800add1a-d459-4796-b974-b2f094db417f",
+   "metadata": {},
+   "source": [
+    "Run next cell to perform another top-level UNION ALL, this time where the sets use `GROUP BY`.\n",
+    "\n",
+    "Notice that the aggregates have `AS` to set specific field names."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8e687466-74bb-4cc0-ba17-913d1807fc60",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql='''\n",
+    "WITH\n",
+    "set1 AS (\n",
+    "  SELECT\n",
+    "    TIME_FLOOR(__time, 'PT1H') AS \"Period\",\n",
+    "    countryName,\n",
+    "    AVG(commentLength) AS \"Average Comment Size\",\n",
+    "    COUNT(DISTINCT \"page\") AS \"Pages\"\n",
+    "  FROM \"example-wikipedia-unionall\"\n",
+    "  WHERE countryName='China'\n",
+    "  GROUP BY 1, 2\n",
+    "  LIMIT 10\n",
+    "  ),\n",
+    "set2 AS (\n",
+    "  SELECT\n",
+    "    TIME_FLOOR(__time, 'PT1H') AS \"Episode\",\n",
+    "    countryName,\n",
+    "    COUNT(DISTINCT \"page\") AS \"Pages\",\n",
+    "    AVG(commentLength) AS \"Average Comment Length\"\n",
+    "  FROM \"example-wikipedia-unionall\"\n",
+    "  WHERE countryName='Austria'\n",
+    "  GROUP BY 1, 2\n",
+    "  LIMIT 10\n",
+    "  )\n",
+    "\n",
+    "SELECT * from set1\n",
+    "UNION ALL\n",
+    "SELECT * from set2\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f2c95ffc-b260-4671-bacc-c8cc3137e9c2",
+   "metadata": {},
+   "source": [
+    "Look carefully at these results - results from `set2` were appended very simply.\n",
+    "\n",
+    "* Column name in `set2` (`Period` versus `Episode` and `Average Comment Size` versus `Average Comment Length`) did not result in new columns\n",
+    "* Columns with the same name (`Pages`) did not result in that aggregate being put into same column - Austria's values are simply appended `Average Comment Size`\n",
+    "\n",
+    "Run the next cell, which uses explicit column names at the top-level, rather than `*`, to ensure the calculations appear in the right columns in the final result. It also explicitly names the column names for the results by using `AS`."
```

Comment:
```suggestion
    "Run the next cell, which uses explicit column names at the top-level, rather than `*`, to ensure the calculations appear in the right columns in the final result. It also aliases the columns for the results by using `AS`."
```
I think this is called "aliasing", but be sure